bors.toml

```diff
@@ -1,6 +1,6 @@
 # The service providing the commit statuses to GitHub.
 status = [
-  "buildbot/ykrustc-buildbot-build-script-hwt",
+  #"buildbot/ykrustc-buildbot-build-script-hwt",
```

Comment:
Doesn't this disable the HWT and not the SWT?

Reply:
Doh. Yes. I see it's going to be one of those days!

Reply:
Fixed, but I should squash.